proknow-sdk/Patient/PatientItem.cs

```diff
@@ -1,348 +1,348 @@
-using ProKnow.Upload;
-using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Net.Http;
-using System.Text;
-using System.Text.Json;
-using System.Text.Json.Serialization;
-using System.Threading.Tasks;
-
-namespace ProKnow.Patient
-{
-    /// <summary>
-    /// Represents a patient in a ProKnow workspace
-    /// </summary>
-    public class PatientItem
-    {
-        private ProKnowApi _proKnow;
-
-        /// <summary>
-        /// The patient workspace ID
-        /// </summary>
-        [JsonIgnore]
-        public string WorkspaceId { get; internal set; }
-
-        /// <summary>
-        /// The patient ProKnow ID
-        /// </summary>
-        [JsonPropertyName("id")]
-        public string Id { get; set; }
-
-        /// <summary>
-        /// The patient medical record number (MRN) or ID
-        /// </summary>
-        [JsonPropertyName("mrn")]
-        public string Mrn { get; set; }
-
-        /// <summary>
-        /// The patient name
-        /// </summary>
-        [JsonPropertyName("name")]
-        public string Name { get; set; }
-
-        /// <summary>
-        /// The patient birth date in the format "YYYY-MM-DD" or null
-        /// </summary>
-        [JsonPropertyName("birth_date")]
-        public string BirthDate { get; set; }
-
-        /// <summary>
-        /// The patient sex, one of "M", "F", "O" or null
-        /// </summary>
-        [JsonPropertyName("sex")]
-        public string Sex { get; set; }
-
-        /// <summary>
-        /// The patient metadata (custom metrics) referenced by ProKnow ID
-        /// </summary>
-        [JsonPropertyName("metadata")]
-        public Dictionary<string, object> Metadata { get; set; }
-
-        /// <summary>
-        /// Entities within this study
-        /// </summary>
-        [JsonPropertyName("studies")]
-        public IList<StudySummary> Studies { get; set; }
-
-        /// <summary>
-        /// Properties encountered during deserialization without matching members
-        /// </summary>
-        [JsonExtensionData]
-        public Dictionary<string, object> ExtensionData { get; set; }
-
-        //todo--Add Tasks property
-
-        /// <summary>
-        /// Used by deserialization to create patient item
-        /// </summary>
-        public PatientItem()
-        {
-        }
-
-        /// <summary>
-        /// Creates a patient item from its JSON representation
-        /// </summary>
-        /// <param name="proKnow">Root object for interfacing with the ProKnow API</param>
-        /// <param name="workspaceId">ID of the workspace containing the patients</param>
-        /// <param name="json">JSON representation of the patient item</param>
-        internal PatientItem(ProKnowApi proKnow, string workspaceId, string json)
-        {
-            Initialize(proKnow, workspaceId, json);
-        }
-        
-        //todo--Add CreateSpatialRegistrationObject method
-
+﻿using ProKnow.Patient.Entities;

```

Comment:
What happened to make Github think this file changed?

Reply:
Undid change, not sure what happened